plugins/hls-overloaded-record-dot-plugin/src/Ide/Plugin/OverloadedRecordDot.hs

```diff
@@ -18,7 +18,7 @@ import           Control.Monad.IO.Class               (MonadIO, liftIO)
 import           Control.Monad.Trans.Except           (ExceptT)
```

Comment:
missing a cabal file edit?

Reply:
It compiles, so I guess some package is reexporting it for us? Tbh I kind of hate this rexporting business, makes it really unclear what package is providing what.

Reply:
Sorry, I didn't mean on this line, I just meant that in general there were no edits to the overloaded-record-dot plugin whereas there are to all the others

Reply:
Oh right, I completely forgot about bumping the version for this one.